ui/conftest.py

```diff
@@ -15,44 +16,46 @@
     "devnet": "NeonEVM DevNet",
 }
 
+CHROME_TAR_PATH = pathlib.Path(__file__).absolute().parent / "extensions" / "data"
+CHROME_DATA_PATH = pathlib.Path(__file__).absolute().parent.parent / "chrome-data" / uuid.uuid4().hex
+"""CHROME_DATA_PATH is temporary local destination in project to untar chrome data directory and plugins"""
 
-CHROME_EXT_DIR = "extensions/chrome/plugins"
-"""Relative path to Chrome extension source
-"""
 
-CHROME_USER_DATA_DIR = "user_data"
-"""Relative path to Chrome extensions user data
-"""
+@pytest.fixture(scope="session")
+def network(pytestconfig: tp.Any) -> tp.Optional[str]:
```

Comment:
we can't run this tests on night-stand

Reply:
If we set the `network` parameter in environment to the `night-stand` value, this method will return `NEON EVM night-stand` value (see the screenshot below). Currently I set  the neonpass_url parameter in the `envs.json` only for devnet environment, but it's also possible to set it for the `night-stand` as well.
<img width="586" alt="Screenshot 2023-06-26 at 11 05 29" src="https://github.com/neonlabsorg/neon-tests/assets/17324832/0aefb850-0a0d-4b5f-ab8a-8f37f5ec3ccb">
<img width="921" alt="Screenshot 2023-06-26 at 11 28 28" src="https://github.com/neonlabsorg/neon-tests/assets/17324832/6021f16b-ae66-452e-b5e4-33e6ad0f227f">